static/src/javascripts/projects/common/modules/navigation/new-header.js

```diff
@@ -320,6 +337,30 @@ const enhanceCheckbox = (checkbox: HTMLElement): void => {
                 button.setAttribute('aria-controls', checkboxControls);
             }
 
+            if (label) {
+                label.classList.forEach(className => {
+                    button.classList.add(className);
+                });
+
+                const labelTabIndex = label.getAttribute('tabindex');
+
+                if (labelTabIndex) {
+                    button.setAttribute('tabindex', labelTabIndex);
+                }
+
+                if (checkboxId === EDITION_PICKER_TOGGLE_CLASS) {
+                    const displayName = label.getAttribute('data-display-name');
+
+                    if (displayName) {
+                        button.innerHTML = btnOpts.innerHTML(displayName);
+                    }
+                } else {
+                    button.innerHTML = btnOpts.innerHTML;
+                }
+
+                label.classList.add('u-h');
```

Comment:
How does this sound on a screen reader? Maybe better to delete the label entirely?

Reply:
Not sure how it sounds with a screen reader, but yeah I don't think labels are necessary for `<button>` elements, so i'll remove them now.